draft-irtf-cfrg-opaque.md

```diff
@@ -609,85 +709,92 @@ Steps:
 3. Output (private_key, public_key)
 ~~~
 
-HashToScalar(msg, dst) is as specified in {{I-D.irtf-cfrg-voprf}}, except that dst = "OPAQUE-HashToScalar".
+HashToScalar(msg, dst) is as specified in {{I-D.irtf-cfrg-voprf}},
+except that dst = "OPAQUE-HashToScalar".
```

Comment:
Perhaps remove the "except that dst = "OPAQUE-HashToScalar"", since this is already specified above?